draft-ietf-dnsop-svcb-https.md

```diff
@@ -86,20 +86,11 @@ collection of consistent configuration parameters through the DNS
 
 This document first describes the SVCB RR as a general-purpose resource
 record that can be applied directly and efficiently to a wide range
-of services ({{svcb}}).  The HTTPS RR is then defined as a special
-case of SVCB that improves efficiency and convenience for use with HTTPS
-({{https}}) by avoiding the need for an Attrleaf label {{?Attrleaf=RFC8552}}
-({{httpsnames}}).  Other protocols with similar needs may
-follow the pattern of HTTPS and assign their own
-SVCB-compatible RR types.
-
-All behaviors described as applying to the SVCB RR also apply
-to the HTTPS RR unless explicitly stated otherwise.
-{{https}} describes additional behaviors
-specific to the HTTPS RR.  Apart from {{https}}
-and introductory examples, much of this document refers only to the SVCB RR,
-but those references should be taken to apply to SVCB, HTTPS,
-and any future SVCB-compatible RR types.
+of services ({{svcb}}).  It also describes the rules for defining other
+SVCB-compatible RR types ({{svcb-compatible}}), starting with the HTTPS
+RR type ({{https}}), which provides improved efficiency and convenience
+with HTTPS by avoiding the need for an Attrleaf label {{?Attrleaf=RFC8552}}
```

Comment:
```suggestion
with HTTP by avoiding the need for an Attrleaf label {{?Attrleaf=RFC8552}}
```

Do we also want a normative reference to HTTP here?

Reply:
I don't think we need a reference.